service/src/main/resources/templates/ehr_encounter_statement_to_ehr_composition_template.mustache

```diff
@@ -0,0 +1,32 @@
+<component typeCode="COMP">
+    <ehrComposition classCode="COMPOSITION" moodCode="EVN">
+        <id root="{{encounterStatementId}}" />
+        <code code="24631000000103" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="Residential Home Visit Note">
+            <originalText>Residential home visit note</originalText>
+        </code>
+        <statusCode code="{{status}}" />
+        <effectiveTime>
+            {{&effectiveTime}}
+        </effectiveTime>
+        <availabilityTime value="20091102" />
```

Comment:
From the Mapping spreadsheet we need to populate availability time with period start